data/mods/Magiclysm/Spells/magus.json

```diff
@@ -63,5 +63,107 @@
     "base_casting_time": 600,
     "energy_source": "MANA",
     "base_energy_cost": 350
+  },
+  {
+    "id": "magus_eldritch_blast",
+    "type": "SPELL",
+    "name": "Eldritch Blast",
+    "description": "A blast of concentrated magical power that obliterates a large area.",
+    "valid_targets": [ "hostile", "ground" ],
+    "effect": "projectile_attack",
+    "spell_class": "MAGUS",
```

Comment:
i wanted eldritch blast to be an animist spell

Reply:
Renamed to Mana Blast.